cves/CVE-2013-7024.yml

```diff
@@ -129,7 +135,13 @@ lifetime:
           git log --stat abc..def -- your/file.c
```

Comment:
I scrolled through the vcc and fixes and they seemed good, none of the vccs looked unnecessary